scripts/update_instance_data/pricing-updater/scripts/create_instance_data.go

```diff
@@ -0,0 +1,79 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+const instanceDataOutputDirPath =  "pkg/util/instanceselector/"
+
+var topDir = os.Getenv("TOP_KIP_DIR")
```

Comment:
In other scripts, we used to use `GOPATH` to figure out the repository path.

Reply:
This may be tricky, as we already migrated to go modules, which means you no longer need to keep this repo inside your GOPATH. For example, on my laptop, I keep it in `$HOME/code/elotl/kip`. So I'd prefer to keep it as it is.